builder/.github/workflows/create-release.yml

```diff
@@ -112,4 +112,4 @@ jobs:
         issue_body: |
           Create Release workflow [failed](https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}).
         comment_body: |
-           Another failure occurred: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+          Another failure occurred: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
```

Comment:
I assume this whitespace change is still valid yaml - is the indentation change a problem?

Reply:
This is actually unrelated to the change, but I couldn't resist to fix it anyway (notice the there are 3 more spaces in front of `Another` compared to `comment_body`, everywhere else "one level" is 2 spaces).

Still, it is unrelated, so I can remove it from this PR, if you prefer.

Reply:
yeah, I appreciate the cleanup! I'm happy to keep it in the PR.

I just wanted to make sure that the indentation doesn't matter (which I think it doesn't because it's effectively ignored in a multi-line string).